config.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import "time"
+
+type State int
+
+const (
+	open State = iota
+	closed
+	failure
+)
+
+type Check struct {
+	host  Host
+	state State
+}
+
+type Host struct {
+	Network string `yaml:"network"`
```

Comment:
Maybe protocol would make more sense here?
```suggestion
	Protocol string `yaml:"protocol"`
```

Reply:
I stuck with `network` because that's the naming convention used in `net.Dial`, think it still makes sense to change?

Reply:
Nah, then it's fine. Disregard.